sql_scripts/1_user_and_database_creation.sql

```diff
@@ -0,0 +1,29 @@
+sudo -u postgres psql postgres
+
+CREATE USER admin_1 WITH PASSWORD '12345678';
+
+CREATE DATABASE breweries;
+
+/* Connect to the "breweries" database */
+
+\c breweries;
+
+CREATE SCHEMA IF NOT EXISTS beers;
+
+GRANT CONNECT ON DATABASE breweries TO admin_1;
```

Comment:
Consider creating role with all permissins needed in separate script and then adding your user to this role